esp/scm/ws_espcontrol.ecm

```diff
@@ -30,7 +30,8 @@ ESPresponse [exceptions_inline, nil_remove, http_encode(0)] SetLoggingResponse
     string Message;
 };
 
-ESPservice [ version("1.00"), default_client_version("1.00"), exceptions_inline("./smc_xslt/exceptions.xslt")] WSESPControl
+//no authfeature is performed, however in cpp, there's a check to see if user is superuser
```

Comment:
Yes, that's what I commented, ask for confirmation that the behavior is correct... I'll leave as is.